app/src/main/java/com/wesleyfranks/todo24/ui/completed/CompletedFragment.kt

```diff
@@ -29,4 +33,24 @@ class CompletedFragment : Fragment() {
         return completeView
     }
 
+    override fun onCreateOptionsMenu(menu: Menu, inflater: MenuInflater) {
+        inflater.inflate(R.menu.create_options, menu)
+        super.onCreateOptionsMenu(menu, inflater)
+    }
+
+    override fun onOptionsItemSelected(item: MenuItem): Boolean {
+        return when (item.itemId){
+            R.id.create_options_deleteAllTodos -> {
+                DeleteAllTodos()
+                true
+            }
+            else -> super.onOptionsItemSelected(item)
+        }
+    }
+
+    private fun DeleteAllTodos() {
+        completedViewModel.deleteAllTodos(binding.root.context)
+        Snackbar.make(binding.root,"All toods have been deleted...", Snackbar.LENGTH_LONG).show()
```

Comment:
I think Completed stuff is a bit out of scope for this PR. But there's also a typo on this line for when the time comes.